satpy/enhancements/__init__.py

```diff
@@ -653,3 +653,8 @@ def _jma_true_color_reproduction(img_data, platform=None):
 
     output = da.dot(img_data.T, ccm.T)
     return output.T
+
+
+def no_op(img):
+    """Do not do anything to the image."""
+    return img.data
```

Comment:
I think we still want a test for this. Just to be sure. A simple `assert res == img.data` should do.

Reply:
ok, on it

Reply:
Fixed in b408e65